### PR TITLE
Track call in Mailchimp doc

### PR DESCRIPTION
**Pull request by ishachopra0401**
## What do these changes do?

> Describe the changes you are proposing in this PR.

## Type of documentation

- [ ] New documentation
- [ ] Documentation update
- [ ] Hotfix

### CODE REVIEW

File: docs/destinations/streaming-destinations/mailchimp.mdx [hunk 1]
```diff
@@ -148,6 +148,31 @@ rudderanalytics.identify(
...
+### Supported mappings
+
+The following table lists the mappings between the RudderStack and Mailchimp properties:
+
+| RudderStack property | Mailchimp property | Notes |
+| :--- | :--- |:--- |
+| `event`<br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `name` | Event name must contain 2-30 characters. | 
+| `properties` | `properties` | - |
+| `properties.is_syncing` | `is_syncing` | Set `false` if you want to [trigger targeted automations](https://mailchimp.com/developer/marketing/guides/track-outside-activity-events/#trigger-automations-with-events). |
+| `originalTimestamp` | `occured_at` |  - |
```
Comment (ujjwal-ab) [inline docs/destinations/streaming-destinations/mailchimp.mdx:174]:
```suggestion
| `originalTimestamp` | `occurred_at` |  - |
```

File: docs/destinations/streaming-destinations/mailchimp.mdx [hunk 2]
```diff
@@ -148,6 +148,31 @@ rudderanalytics.identify(
...
+
+You can use the <Link to="/event-spec/standard-events/track">`track`</Link> call to collect website/in-app actions or [trigger targeted automations](https://mailchimp.com/developer/marketing/guides/track-outside-activity-events/#trigger-automations-with-events) in Mailchimp using their [Events](https://mailchimp.com/developer/marketing/api/list-member-events/) API.
+
+A sample `track` call is shown below:
+
+```javascript
+rudderanalytics.track("registered_referral", {
+  "referee_id": "1233",
+  "referral_acceptance": true,
+  "is_syncing": false
```
Comment (ujjwal-ab) [inline docs/destinations/streaming-destinations/mailchimp.mdx:161]:
```suggestion
  "referral_acceptance": true
```

File: docs/destinations/streaming-destinations/mailchimp.mdx [hunk 3]
```diff
@@ -148,6 +148,31 @@ rudderanalytics.identify(
...
+
+### Supported mappings
+
+The following table lists the mappings between the RudderStack and Mailchimp properties:
+
+| RudderStack property | Mailchimp property | Notes |
+| :--- | :--- |:--- |
+| `event`<br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `name` | Event name must contain 2-30 characters. | 
+| `properties` | `properties` | - |
+| `properties.is_syncing` | `is_syncing` | Set `false` if you want to [trigger targeted automations](https://mailchimp.com/developer/marketing/guides/track-outside-activity-events/#trigger-automations-with-events). |
```
Comment (ujjwal-ab) [inline docs/destinations/streaming-destinations/mailchimp.mdx:173]:
```suggestion
| `properties.is_syncing` | `is_syncing` | Set `true` if you don't want to [trigger targeted automations](https://mailchimp.com/developer/marketing/guides/track-outside-activity-events/#trigger-automations-with-events). Triggering targeted automation is the by default behaviour. |
```

**Review by ujjwal-ab — APPROVED** [review]

File: docs/destinations/streaming-destinations/mailchimp.mdx [hunk 4]
```diff
@@ -148,6 +148,35 @@ rudderanalytics.identify(
...
+```javascript
+rudderanalytics.track("registered_referral", {
+  "referee_id": "1233",
+  "referral_acceptance": true,
+  "email": "alex@example.com"
+})
+```
+
+<div class="infoBlock">
+<code class="inline-code">email</code> is a required field for making a <code class="inline-code">track</code> call successfully and can be passed inside <code class="inline-code">traits.email</code> , <code class="inline-code">context.traits.email</code>, or <code class="inline-code">properties.email</code>. RudderStack hashes the <code class="inline-code">email</code> field and replaces <code class="inline-code">subscriber_hash</code> in the <a href="https://mailchimp.com/developer/marketing/api/list-member-events/">Mailchimp Events API endpoint</a>.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mailchimp.mdx:166]:
```suggestion
<code class="inline-code">email</code> is a required field for making a <code class="inline-code">track</code> call successfully. You can pass it in either <code class="inline-code">traits.email</code> , <code class="inline-code">context.traits.email</code>, or <code class="inline-code">properties.email</code>. RudderStack hashes the <code class="inline-code">email</code> field and replaces <code class="inline-code">subscriber_hash</code> in the <a href="https://mailchimp.com/developer/marketing/api/list-member-events/">Mailchimp Events API endpoint</a>.
```

File: docs/destinations/streaming-destinations/mailchimp.mdx [hunk 5]
```diff
@@ -148,6 +148,35 @@ rudderanalytics.identify(
 This setting will <strong>NOT</strong> work for new users. For new users, the subscription status will always be set as <code class="inline-code">pending</code> or <code class="inline-code">subscribed</code>, depending on the set <code class="inline-code">doubleOptIn</code> setting.
 </div>
 
+## Track
+
+You can use the <Link to="/event-spec/standard-events/track">`track`</Link> call to collect website/in-app actions or [trigger targeted automations](https://mailchimp.com/developer/marketing/guides/track-outside-activity-events/#trigger-automations-with-events) in Mailchimp using their [Events](https://mailchimp.com/developer/marketing/api/list-member-events/) API.
```
Comment (ameypv-rudder) [inline docs/destinations/streaming-destinations/mailchimp.mdx:153]:
```suggestion
You can use the <Link to="/event-spec/standard-events/track">`track`</Link> call to track user actions or [trigger targeted automations](https://mailchimp.com/developer/marketing/guides/track-outside-activity-events/#trigger-automations-with-events) in Mailchimp using their [Events](https://mailchimp.com/developer/marketing/api/list-member-events/) API.
```

Comment (ameypv-rudder) [reply docs/destinations/streaming-destinations/mailchimp.mdx:153]: Simplified the idea - lmk if it makes sense.

**Review by ameypv-rudder — APPROVED** [review]

File: docs/destinations/streaming-destinations/mailchimp.mdx [hunk 6]
```diff
@@ -148,6 +148,35 @@ rudderanalytics.identify(
...
+## Track
+
+You can use the <Link to="/event-spec/standard-events/track">`track`</Link> call to track user actions or [trigger targeted automations](https://mailchimp.com/developer/marketing/guides/track-outside-activity-events/#trigger-automations-with-events) in Mailchimp using their [Events](https://mailchimp.com/developer/marketing/api/list-member-events/) API.
+
+A sample `track` call is shown below:
+
+```javascript
+rudderanalytics.track("registered_referral", {
+  "referee_id": "1233",
+  "referral_acceptance": true,
```
Comment (ujjwal-ab) [inline docs/destinations/streaming-destinations/mailchimp.mdx:160]:
```suggestion
  "referral_acceptance": "accepted",
```

File: docs/destinations/streaming-destinations/mailchimp.mdx [hunk 7]
```diff
@@ -148,6 +148,35 @@ rudderanalytics.identify(
...
+
+### Supported mappings
+
+The following table lists the mappings between the RudderStack and Mailchimp properties:
+
+| RudderStack property | Mailchimp property | Notes |
+| :--- | :--- |:--- |
+| `event`<br/> <span style="color: #4D4DFF;font-size:12px;">Required</span> | `name` | Event name must contain 2-30 characters. | 
+| `properties` | `properties` | - |
+| `properties.is_syncing` | `is_syncing` | RudderStack triggers the [targeted automations](https://mailchimp.com/developer/marketing/guides/track-outside-activity-events/#trigger-automations-with-events) by default. However, you can set `is_syncing` to `true` if you don't want to trigger them. |
```
Comment (ujjwal-ab) [inline docs/destinations/streaming-destinations/mailchimp.mdx:177]:
```suggestion
| `properties.isSyncing` | `is_syncing` | RudderStack triggers the [targeted automations](https://mailchimp.com/developer/marketing/guides/track-outside-activity-events/#trigger-automations-with-events) by default. However, you can set `isSyncing` to `true` if you don't want to trigger them. |
```

**Review by ujjwal-ab — COMMENTED** [review]
LGTM

**Pull request merged**